source/common/access_log/access_log_formatter.cc

```diff
@@ -125,8 +127,10 @@ JsonFormatterImpl::JsonFormatterImpl(std::unordered_map<std::string, std::string
 std::string JsonFormatterImpl::format(const Http::HeaderMap& request_headers,
                                       const Http::HeaderMap& response_headers,
                                       const Http::HeaderMap& response_trailers,
-                                      const StreamInfo::StreamInfo& stream_info) const {
-  const auto output_map = toMap(request_headers, response_headers, response_trailers, stream_info);
+                                      const StreamInfo::StreamInfo& stream_info,
+                                      const absl::string_view& body) const {
```

Comment:
This only applies to locally generated responses, right? We don't have to worry about the remote having a 100MB response? What if someone specifies RESP_BODY on a regular access log formatter?

Reply:
Body is passed only for `local_reply`, for access log I'm passing empty string so this shouldn't be a problem.

Reply:
I think the direct response route action also goes through `sendLocalReply`. So, keep in mind that potentially largeish files could be present.